.github/workflows/node.js.yml

```diff
@@ -26,7 +26,7 @@ jobs:
            npm install nyc coveralls bslint @hns-dev/bsdoc
```

Comment:
We may want to use specific versions here, otherwise in the future CI lint for these may break. E.g. nyc or coveralls added new dep that has disallowed characters.